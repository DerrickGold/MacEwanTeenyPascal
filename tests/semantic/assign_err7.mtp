(*try to index into a variable*)

var notArray : integer;

begin
	notArray(0) := 1
end.