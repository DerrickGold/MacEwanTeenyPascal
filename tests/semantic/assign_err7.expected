
Semantic Error [line 6]: value: "notArray"
	Index attempt on non-array variable.
